src/plugins/comicsPlayer/plugin.js

```diff
@@ -96,11 +157,71 @@ export class ComicsPlayer {
         }
     }
 
+    //Natively available in v8.2.6 [https://github.com/nolimits4web/Swiper/issues/3279]
```

Comment:
Related to #3892

Reply:
I did try to update the package, but the build was failing.

Reply:
Successfully fixed the build, added the changes in this PR